lib/addic7ed/subtitle.rb

```diff
@@ -53,10 +55,22 @@ def normalize_version!
       @version.upcase!
     end
 
+    def normalize_comment!
+      @comment ||= ""
+      @comment.downcase!
+    end
+
     def is_compatible_with?(other_version)
-      self.version == other_version ||
+      version_compatible = self.version == other_version ||
       COMPATIBILITY_720P[self.version] == other_version ||
       COMPATIBILITY_720P[other_version] == self.version
+
+      comment_compatible = self.comment.include? other_version.downcase
+      if COMPATIBILITY_720P[other_version]
+        comment_compatible ||= self.comment.include? COMPATIBILITY_720P[other_version].downcase
```

Comment:
You can use `COMPATIBILITY_720P.map(&:downcase)[other_version]` instead of a big `if COMPATIBILITY_720P[other_version]`, here.